demisto_sdk/commands/validate/validators/BA_validators/BA108_is_folder_name_has_separators.py

```diff
@@ -0,0 +1,35 @@
+from __future__ import annotations
+
+from typing import Iterable, List, Union
+
+from demisto_sdk.commands.common.constants import GitStatuses
+from demisto_sdk.commands.content_graph.objects.integration import Integration
+from demisto_sdk.commands.content_graph.objects.script import Script
+from demisto_sdk.commands.validate.validators.base_validator import (
+    BaseValidator,
+    ValidationResult,
+)
+
+ContentTypes = Union[Integration, Script]
+
+
+class IsFolderNameHasSeparatorsValidator(BaseValidator[ContentTypes]):
+    error_code = "BA108"
+    description = "Check if there are separators in the folder name."
+    error_message = "The folder name '{0}' should be without any separator."
```

Comment:
```suggestion
    error_message = "The folder name '{0}' should be without any separator."
```
Let's add an example of the separators we're talking about.